src/backend/distributed/citus--7.0-1--7.0-2.sql

```diff
@@ -0,0 +1,86 @@
+/* citus--7.0-1--7.0-2.sql */
+
```

Comment:
Maybe that should be a separate PR? They seem super easy to rebase onto each other I can open another PR for it soon.